src/user-management/user-management.ts

```diff
@@ -155,19 +156,28 @@ const toQueryString = (options: Record<string, string | undefined>): string => {
 
 export class UserManagement {
   private jwks: ReturnType<typeof createRemoteJWKSet> | undefined;
+  public clientId: string | undefined;
+  public ironSessionProvider: IronSessionProvider;
 
   constructor(
     private readonly workos: WorkOS,
-    private readonly ironSessionProvider: IronSessionProvider,
+    ironSessionProvider: IronSessionProvider,
   ) {
     const { clientId } = workos.options;
 
+    this.clientId = clientId;
+    this.ironSessionProvider = ironSessionProvider;
+
     // Set the JWKS URL. This is used to verify if the JWT is still valid
     this.jwks = clientId
       ? createRemoteJWKSet(new URL(this.getJwksUrl(clientId)))
       : undefined;
   }
 
+  session(sessionData: string, cookiePassword: string): Session {
+    return new Session(this, sessionData, cookiePassword);
+  }
```

Comment:
I wonder if this should be called something like `sessionFromCookie()`. Also do we want to allow `WORKOS_COOKIE_PASSWORD` from environment variable at all? We currently do this, but I don't have a strong opinion either way.

Reply:
I kinda like the terseness of `session()`.

I want to shy away from using environment variables as it causes headaches in environments that don't support `process` (looking at you CFW).

Reply:
Good point on CFW, works for me!